cloud/src/common/config.h

```diff
@@ -263,7 +263,7 @@ CONF_Bool(enable_check_instance_id, "true");
 CONF_Bool(enable_loopback_address_for_ms, "false");
 
 // delete_bitmap_lock version config
-CONF_mString(use_delete_bitmap_lock_version, "v1");
+CONF_mString(delete_bitmap_lock_v2_white_list, "");
```

Comment:
Add a note in the comment explaining the usage of this configuration.

Reply:
done